  RuntimeError() : std::runtime_error("") {}